test/v2/unit/coins/ltc.js

```diff
@@ -120,19 +119,21 @@ describe('LTC:', function() {
     });
 
     it('should generate p2sh-wrapped segwit address', () => {
-      const generatedAddress = coin.generateAddress({ keychains, addressType: AbstractUtxoCoin.AddressTypes.P2SH_P2WSH });
-      const generatedTestAddress = testCoin.generateAddress({ keychains, addressType: AbstractUtxoCoin.AddressTypes.P2SH_P2WSH });
+      const addressType = Codes.UnspentTypeTcomb('p2shP2wsh');
+      const chain = Codes.forType(addressType)[Codes.PurposeTcomb('external')];
+      const generatedAddress = coin.generateAddress({ keychains, addressType, chain });
+      const generatedTestAddress = testCoin.generateAddress({ keychains, addressType, chain });
 
       [generatedAddress, generatedTestAddress].forEach((currentAddress) => {
-        currentAddress.chain.should.equal(0);
+        currentAddress.chain.should.equal(chain);
         currentAddress.index.should.equal(0);
-        currentAddress.coinSpecific.outputScript.should.equal('a91426e34781478f08fff903cb70ae67311c3f9bc6a987');
-        currentAddress.coinSpecific.redeemScript.should.equal('00209a10eb58331e95333f4a6eafd5f03e442e17e0986c824e392642e872f431b7ef');
-        currentAddress.coinSpecific.witnessScript.should.equal('5221037acffd52bb7c39a4ac3d4c01af33ce0367afec45347e332edca63a38d1fb2e472102658831a87322b3583515ca8725841335505755ada53ee133c70a6b4b8d3978702102641ee6557561c9038242cafa7f538070d7646a969bcf6169f9950abfcfefd6b853ae');
+        currentAddress.coinSpecific.outputScript.should.equal('a9147ff13f3faeba4d439ef40604f7c127951e77eb6a87');
+        currentAddress.coinSpecific.redeemScript.should.equal('00207aad7d57b238a09b5daa10ff47c54483b7f2ad47f3f0c0aa230958b9df334260');
```

Comment:
These scripts are changing because the chain code is changing. With these changes I can not change the inputs such that the scripts stay the same because the tests generated a p2shP2wsh address on chain 0, which will now throw with these changes due to a mismatch in address type and chain code.